critiquebrainz/frontend/external/musicbrainz_db/tests/release_group_test.py

```diff
@@ -12,9 +12,10 @@ def setUp(self):
         setup_cache()
         mb_release_group.mb_session = MagicMock()
         self.mock_db = mb_release_group.mb_session.return_value.__enter__.return_value
-        self.release_group_query = self.mock_db.query.return_value.options.return_value.\
-                options.return_value.options.return_value.options.return_value.filter.return_value.all
-        self.release_group_query_without_artists = self.mock_db.query.return_value.options.return_value.filter.return_value.all
+        self.release_group_query = self.mock_db.query.return_value.options.return_value.options.return_value.\
+            options.return_value.options.return_value.options.return_value.filter.return_value.all
+        self.release_group_query_without_artists = self.mock_db.query.return_value.\
+            options.return_value.options.return_value.filter.return_value.all
```

Comment:
Can you please decipher this? I have no idea what value comes out of this or why it has to be retrieved like that?